client/client.go

```diff
@@ -33,14 +33,14 @@ const (
 )
 
 func init() {
-	data.SetDefaultExpiryTimes(notary.NotaryDefaultExpiries)
+	data.SetDefaultExpiryTimes(data.NotaryDefaultExpiries)
```

Comment:
@endophage I moved `NotaryDefaultExpiries` from the `notary` package to the `data` package to avoid a cyclic dependency, as `NotaryDefaultExpiries` is now using the `RoleName` type as the map key. tuf/data/types.go imports `github.com/docker/notary` so I could not import `github.com/docker/notary/tuf/data` into const.go (`notary` package) without a cyclic dependency.

https://github.com/docker/notary/pull/1087/files#diff-9d6b1e1df9b74f1f173c502269aa1ae3L89
https://github.com/docker/notary/pull/1087/files#diff-de0c95529058d827a3e5143442b9de1cR69